src/generate-common-seo.js

```diff
@@ -48,10 +48,13 @@ export function generateStructuredData(config = {}) {
   const { "theme-attributes":themeConfig, "social-links":socialLinks, "seo-metadata":seoMetadata = [] } = config;
   const homePageSeo = seoMetadata.find(page => page["owner-type"] === "home") || {};
   const { "page-title":pageTitle = "", description = "", keywords  =  "" } = get(homePageSeo, ["data"], {});
-
+  let enableNewsArticle = themeConfig['structured_data_news_article'];
   if(!themeConfig || !themeConfig.logo) {
     return {};
   }
+  if(config.hasOwnProperty('enableNewsArticle')){
```

Comment:
`hasOwnProperty` will return true if the value exists, even if it exists as `undefined`. Better to use the simpler syntax.

```suggestion
  if(config.enableNewsArticle){
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#note

Reply:
My bad, this won't work because we need the condition to pass if the value is `false` as well.

There are a lot of hacks before and after this flow for that value, so for now `hasOwnProperty` along with a check for undefined should be fine.